src/v0/util/cache.js

```diff
@@ -42,6 +51,7 @@ class Cache {
     // Check if key exists in cache (use has() to handle falsy values correctly)
     if (this.cache.has(key)) {
       const value = this.cache.get(key);
+      this.emitStats();
```

Comment:
Shouldn't we emit stats in line 60 too?

Reply:
`return undefined;` Lint 60 is this. Do you mean this?

Reply:
Yes. Why emit stats only for `this.cache.has(key)` case?

Reply:
We are emiting stats when some actions is being done. Here in this particular case it's for `.get` case and not for `.has`. For set and del we are already doing it via event handlers.

Reply:
Shashank has a point. Since, we're interested in cache miss stats, even in this scenario, we should emit metrics.